massgov/mayflower
# DP-11158:  Downloads page.

## Description
This adds/changes a few things that were missing from the ListingTable component:
* Adds ListingTable to the index.js file.
* Fixes unique key issues on lists.
* Adds tbody tag to table.
* Fixes PropTypes definition for rows prop.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-11158)

## Steps to Test
1. Ensure that there are no longer any key warnings in console when viewing the ListingTable story in storybook.
2. Ensure there's no warnings about PropTypes for rows when viewing the ListingTable story.
